releases/foreman/settings

```diff
@@ -2,3 +2,4 @@ TAR_PROJECTS="foreman foreman-proxy foreman-installer foreman-selinux"
 RPM_PACKAGES="foreman foreman-installer foreman-proxy foreman-release foreman-selinux"
 KOJI_TAGS="foreman-@VERSION@-@OS@"
 PACKAGING_SUBDIR="packages/foreman"
+OSES="el8"
```

Comment:
I was hesitant to change it in `settings`. Perhaps we should drop it altogether there and enforce all projects to set it explicitly?

Reply:
sounds fair to me